### DISCUSSION TITLE
画质/清晰度选择

### DISCUSSION THREAD

**Discussion by wzdc** [AlistGo/alist#4246] [Ideas]
（在线播放视频）
希望pikpak也可以像阿里云盘那样可以选择画质

**Comment by anwen-anyi** [comment]
阿里云盘可以选择清晰度那样是因为接入了阿里云盘OpenAPI接口 用官方的API实现的😝

**Reply by wzdc** [reply]
![image](https://user-images.githubusercontent.com/53571020/234573159-cb037667-716b-4926-b2af-79f1b49727e9.png)
pikpak也有接口可以获取清晰度呀

**Reply by anwen-anyi** [reply]
> ![image](https://user-images.githubusercontent.com/53571020/234573159-cb037667-716b-4926-b2af-79f1b49727e9.png)
> 
> pikpak也有接口可以获取清晰度呀

😂你指的分享还是Pikpak个人的 个人的好像是有 不过分享应该不行?

**Reply by wzdc** [reply]
个人的
